data/staff.json

```diff
@@ -1,42 +1,82 @@
 [
-  {
-    "user": "Jriwanek",
-    "icon": "https://mcmoddev.com/img/jriwanek.jpg",
-    "role": "Mod Developer | MMD Admin"
+	{
+		"user": "Jriwanek",
+		"icon": "https://mcmoddev.com/img/jriwanek.jpg",
+		"role": "Mod Developer | MMD Admin"
+	},
+	{ 
+		"user": "Darkhax",
+		"icon": "https://mcmoddev.com/img/darkhax.png",
+		"role": "Mod Developer | MMD Admin"

```

Comment:
I actually don't work on any of the MMD mods directly. (unless you count all of my mods, since they're technically diluv mods as well). I would prefer to be listed as `MMD  Admin | Diluv though.`